src/manifest-webapp-6-2018.json

```diff
@@ -16,5 +16,5 @@
   "theme_color": "black",
   "background_color": "black",
   "display": "standalone",
-  "start_url": "/index.html?utm_source=homescreen"
```

Comment:
In GA I'm seeing a lot of users hitting index.html, I'm guessing now it's because they're primarily PWA users. It's kind of nice to see them split out, so now I'm thinking I may actually reverse this.

Reply:
Well the idea was to use the `utm_source` parameter to distinguish PWA users